pkg/injector/injector.go

```diff
@@ -86,12 +87,21 @@ func NewInjector(config Config, daprClient scheme.Interface, kubeClient *kuberne
 		},
 		kubeClient: kubeClient,
 		daprClient: daprClient,
+		authUID:    authUID,
 	}
 
 	mux.HandleFunc("/mutate", i.handleRequest)
 	return i
 }
 
+func ReplicasetAccountUID(kubeClient *kubernetes.Clientset) (string, error) {
```

Comment:
nit: ReplicaSetAccountUID and comment, linter will complain :-)

Reply:
Didn't complain